packages/rum/src/domain/record/mutationObserver.spec.ts

```diff
@@ -431,6 +447,93 @@ describe('startMutationCollection', () => {
         ],
       })
     })
+
+    describe('for shadow DOM', () => {
+      it('should call shadowDomCreatedCallback when host is added', () => {
+        updateExperimentalFeatures(['recordShadowDom'])
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏use `snake_case` for flag names